cardano-node/src/Cardano/Common/Parsers.hs

```diff
@@ -104,6 +148,21 @@ parseTopologyFile =
          <> metavar "FILEPATH"
          <> help "The path to a file describing the topology."
     )
+parseLogConfigFile :: Parser FilePath
+parseLogConfigFile =
+  strOption
+    ( long "log-config"
+    <> metavar "LOGCONFIG"
+    <> help "Configuration file for logging"
+    <> completer (bashCompleter "file")
+    )
+
+parseLogMetrics :: Parser (Last Bool)
+parseLogMetrics =
+  (Last . Just) <$> switch
```

Comment:
Existing functions like `lastBoolOption`? Formatting? :pray:

Reply:
@ksaric I will clean up and condense all the of `Last` parsing functions in a follow up PR.